src/commerce/types.ts

```diff
@@ -12,6 +12,8 @@ interface CommerceItem {
 }
 
 interface CommerceUser {
+  userId?: string;
+  email?: string;
```

Comment:
shouldn't need email or userId, network interceptor should add those in.

Reply:
Done